composer/yamls/algorithms/fused_layernorm.yaml

```diff
@@ -0,0 +1,2 @@
+# Copyright 2022 MosaicML Composer authors
```

Comment:
if this is empty, is it needed?

Reply:
The existence of the YAML file is needed for things like `test_alg_yamls`, since it finds the algorithms to test (and their default hyperparameters) from there. This file happens to be empty because FLN doesn't have any hyperparameters.